apps/next-evans/lib/env.ts

```diff
@@ -0,0 +1,5 @@
+import invariant from 'tiny-invariant';
+
+invariant(process.env.NEXT_PUBLIC_LANG, `process.env.NEXT_PUBLIC_LANG is not defined`);
+
+export const LANG = process.env.NEXT_PUBLIC_LANG === `es` ? `es` : `en`;
```

Comment:
this lib file will be where we put any environment variables, so that we can import them knowing that they exist, and match the typescript types.